deploy/template.yaml

```diff
@@ -126,7 +110,7 @@ Resources:
           EXPERIAN_API_WRAPPER_SAA_RESOURCE: /question-request
           EXPERIAN_API_WRAPPER_URL: https://zn3dus332j.execute-api.eu-west-2.amazonaws.com/dev
```

Comment:
This should ideally be passed as a parameter, either from the parameter store, or via the samconfig.toml file

Reply:
Agreed. It is part of another story - https://govukverify.atlassian.net/browse/KBV-245